tests/agent/nanojava_interpreter_test.cc

```diff
@@ -0,0 +1,3035 @@
+/**
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "src/agent/nanojava_interpreter.h"
+
+#include <cstdint>
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+#include "jni_proxy_classcastexception.h"
+#include "jni_proxy_classfiletextifier.h"
+#include "jni_proxy_exception.h"
+#include "jni_proxy_jasmin_main.h"
+#include "jni_proxy_object.h"
+#include "jni_proxy_string.h"
+#include "src/agent/jni_method_caller.h"
+#include "src/agent/jvm_class_indexer.h"
+#include "src/agent/model_util.h"
+#include "src/agent/type_util.h"
+#include "tests/agent/jasmin_utils.h"
+
+using testing::MatchesRegex;
+
+namespace {
+
+// Provides absl::Substitue type functionality, but only accepts strings as
```

Comment:
Nit: Typo.

Ordinarily I'd want to tie a comment like this to a bug/issue, but this should be fine as is because I don't see the value in migrating from glog and gflags to abseil at this point.  I assume you agree.

Reply:
Agreed.